src/it/resources/org/checkstyle/suppressionxpathfilter/throwscount/SuppressionXpathRegressionThrowsCount1.java

```diff
@@ -0,0 +1,24 @@
+package org.checkstyle.suppressionxpathfilter.throwscount;
+
+public class SuppressionXpathRegressionThrowsCount1 {
+    public void myFunction() throws CloneNotSupportedException,
+            ArrayIndexOutOfBoundsException,
+            StringIndexOutOfBoundsException,
+            IllegalStateException,
+            NullPointerException { // warn, max allowed is 4
```

Comment:
```
[ERROR] testOne  Time elapsed: 0.039 s  <<< FAILURE!
com.google.common.truth.AssertionErrorWithFacts: 
input file is expected to have a warning comment on line number 4
expected to be true
```
it expects a warning comment `// warn, max allowed is 4` on line 4, however, you put it on line 8. Such comments have to match with actual warnings.

Reply:
done